docs/content/docs/recipes/teams.md

```diff
@@ -118,15 +118,15 @@ export async function getContainer(id : string) : Promise<IFluidContainer> {
 
 ### Creating Fluid container in the configuration page
 
-Open the file `src/client/<your tab name>/<your tab name>Config.tsx` in your code editor. The standard Teams tab application flow goes from configuration to content page. To allow collaborations, we need to find a way to persist the container while loading into the content page. The best solution to persist the container is to append the container ID onto the `contentUrl` and `websiteUrl`, the URLs of the content page, as a query parameter. Given that the save button in the Teams configuration page is the gateway between configuration page and content page, it is a great place for us to create the container and append the container ID in the settings.
+Open the file `src/client/<your tab name>/<your tab name>Config.tsx` in your code editor. The standard Teams tab application flow goes from configuration to content page. To allow collaborations, persisting the container while loading into the content page is crucial. The best solution to persist the container is to append the container ID onto the `contentUrl` and `websiteUrl`, the URLs of the content page, as a query parameter. Given that the save button in the Teams configuration page is the gateway between configuration page and content page, it is a great place to create the container and append the container ID in the settings.
 
 1. Add the following import statement.
 
 ```ts
 import { createContainer, containerIdQueryParamKey } from "./Util";
 ```
 
-1. Replace the `onSaveHandler` method with the following code. Note that the only lines we added are calling the create container method we defined earlier. Then we append the returned container ID in the `contentUrl` and `websiteUrl` as a query parameter.
+1. Replace the `onSaveHandler` method with the following code. Note that the only lines added here are calling the create container method defined earlier in `Utils.ts`. Then appending the returned container ID to the `contentUrl` and `websiteUrl` as a query parameter.
```

Comment:
Can you highlight the code that was added? See [the docs system docs](https://fluidframework.com/community/doc-system#syntax-formatting-and-line-highlighting) for more info on the syntax.

Reply:
learned something new 😀

Reply:
```suggestion
1. Replace the `onSaveHandler` method with the following code. Note that the only lines added here are calling the create container method defined earlier in `Utils.ts` and then appending the returned container ID to the `contentUrl` and `websiteUrl` as a query parameter.
```
hijacking this comment because it seems like github won't let me make another comment on the same lines as this one??

Reply:
no worries 😀